library/patient.inc

```diff
@@ -812,6 +814,15 @@ function getPatientPID($args)
 
     return $returnval;
 }
+function getPatientDoBonly($pid)
```

Comment:
Don't forget to add a function docblock. I know we have hardly *any* docblocks, but if we're bringing in new code it's really something we should do

```
/**
 * Get date of birth for given patient.
 *
 * @param $pid int PID of patient
 * @return DateTime (Or whatever you end up returning
 */